src/accelerate/commands/config/cluster.py

```diff
@@ -16,22 +16,22 @@
 
 from accelerate.state import ComputeEnvironment, DistributedType
 
-from .config_args import ClusterConfig
+from .config_args import ClusterConfig, DeepSpeedConfig
 from .config_utils import _ask_field, _convert_distributed_mode, _convert_yes_no_to_bool
 
 
 def get_cluster_input():
     distributed_type = _ask_field(
-        "Which type of machine are you using? ([0] No distributed training, [1] multi-CPU, [2] multi-GPU, [3] TPU): ",
+        "Which type of machine are you using? ([0] No distributed training, [1] multi-CPU, [2] multi-GPU, [3] DeepSpeed [4] TPU): ",
         _convert_distributed_mode,
-        error_message="Please enter 0, 1, 2 or 3.",
+        error_message="Please enter 0, 1, 2, 3 or 4.",
```

Comment:
I think deepspeed is a subcase of multi-CPU/multi-GPU no? I would rather ask the question after.